oaknational/Oak-Web-Application
# fix: 789 focus element transparency and color

## Description

- fixes color/transparency on "current" link buttons

## Issue(s)

Fixes #789

## How to test

1. Go to {cloud link}
2. make screen small like a phone
3. Go to "about" nav links
4. tab thorugh them
5. focus underline should **not** be transparent


## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
